dp.html

```diff
@@ -0,0 +1,103 @@
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/js/all.min.js" integrity="sha512-fD9DI5bZwQxOi7MhYWnnNPlvXdp/2Pj3XSTRrFs5FQa4mizyGLnJcN6tuvUS6LbmgN1ut+XGSABKvjN0H6Aoow==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+        <!-- Google fonts-->
+        <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css" />
+        <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700" rel="stylesheet" type="text/css" />
+        <!-- Core theme CSS (includes Bootstrap)-->
+        <link href="css/styles.css" rel="stylesheet" />
+        <!-- Custom CSS -->
+        <style>
+            /* Réduire la largeur des champs */
+            .custom-form-control {
+                max-width: 450px;
+            }
+        </style>
+    </head>
+<body id="page-top">
+<!-- Navigation-->
+<nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
+    <div class="container">
+        <a class="navbar-brand" href="./index.html#page-top"><h3>LiberaGroup</h3></a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+            Menu
+            <i class="fas fa-bars ms-1"></i>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav text-uppercase ms-auto py-4 py-lg-0">
+                <li class="nav-item"><a class="nav-link" href="./index.html#services">Notre but</a></li>
+                <li class="nav-item"><a class="nav-link" href="./index.html#about">À propos</a></li>
+                <li class="nav-item"><a class="nav-link" href="./index.html#team">Notre équipe</a></li>
+                <li class="nav-item"><a class="nav-link" href="./index.html#contact">Nous contacter</a></li>
+                <li class="nav-item"><a class="nav-link" href="/redirect/liberadev.html">LiberaDev</a></li>
+                <li class="nav-item"><a class="nav-link" href="/redirect/liberaedu.html">LiberaEdu</a></li>
+            </ul>
+        </div>
+    </div>
+</nav>
+<!-- Masthead-->
+<header class="masthead">
+    <div class="container">
+        <div class="masthead-subheading">LiberaGroup Protection des Données</div>
+        <div class="masthead-heading text-uppercase">Besoin de contacter notre équipe ?</div>
+        <a class="btn btn-primary btn-xl text-uppercase" href="#contact">Nous sommes là pour ça</a>
+    </div>
+</header>
+<!-- Contact-->
+<section class="page-section d-flex justify-content-center align-items-center" id="contact">
+    <div class="container">
+        <div class="text-center">
+            <h2 class="section-heading text-uppercase">Nous contacter</h2>
+            <h3 class="section-subheading text-muted">Une question ? Une proposition ? Juste en-dessous ça passe.</h3>
```

Comment:
In this form you don't send a mail for send your "proposition", why send a proposition ?!

Reply:
Change the "Juste en-dessous ça passe.": "Cela se passe juste en dessous."